packages/amplify-nodejs-runtime-provider/src/utils/legacyBuild.ts

```diff
@@ -10,9 +10,7 @@ export async function buildResource(request: BuildRequest): Promise<BuildResult>
 
   if (!request.lastBuildTimestamp || isBuildStale(request.srcRoot, request.lastBuildTimestamp)) {
     installDependencies(resourceDir);
-    if (request.legacyBuildHookParams) {
-      runBuildScriptHook(request.legacyBuildHookParams.resourceName, request.legacyBuildHookParams.projectRoot);
-    }
+    runBuildScriptHook(request.legacyBuildHookParams.resourceName, request.legacyBuildHookParams.projectRoot);
```

Comment:
Is this tied to the python provider or the platform?

Reply:
It's the platform but it won't break anything to revert